lib/components/artist/artworks/grid.js

```diff
@@ -36,8 +37,9 @@ class ArtworksGrid extends React.Component {
     page: number,
     completed: boolean,
     fetchingNextPage: boolean
-  };
-  _sentEndForContentLength: null | number;
+  }
```

Comment:
`_sentEndForContentLength: null | number;` the `;` is optional, but if it's in a list, it needs commas